### ISSUE TITLE
Change tabs on hover setting checkbox has no affect

### ISSUE THREAD

**Issue by sjclayton** [zen-browser/desktop#1823]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

The option `Change tabs on hover` in Look and Feel > Split View seems to be always getting enabled whether or not the box is checked.

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.6

### What platform are you seeing the problem on?

Linux

**Comment by AryanDesh** [comment]
i have a windows device and this isnt an issue for me, could it be a linux specific issue?

**Comment by sjclayton** [comment]
> could it be a linux specific issue?

It might be specific to Linux *shrugs*

**Comment by gomiMeru** [comment]
I´m on windows and, at least in computer, the checkbox also doesn't seem to have any effect

**Comment by AryanDesh** [comment]
It seems to be working just fine for me, but i found out the close split view doesnt work for me

**Comment by wellenfunktion** [comment]
> i have a windows device and this isnt an issue for me, could it be a linux specific issue?

Doesn't work for me, both on Windows and on Linux. I don't think it's a Linux specific issue.

**Comment by sjclayton** [comment]
@guigirl42  

Because enabling / disabling it right now makes no difference as the feature is stuck always on, hence why this issue got opened.🤪

If you have tabs in split view, mouse over back and forth between them and you will see Zen changing the currently focused tab, this is the behaviour that is supposed to happen only if the setting is enabled.

**Comment by guigirl42** [comment]
Ta, yes, that's exactly what i assumed, but as i stated, no such behaviour occurs for me, hence i assumed i must have misunderstood its intention.

**Comment by rubaboo** [comment]
Same. Pretty major issue in my view, as I have to carefully guide my mouse around the perimeter if I want to click Back/Forward buttons for the tab on the right.
And "effect" was correct btw. Cheers.

**Comment by wellenfunktion** [comment]
> Same. Pretty major issue in my view, as I have to carefully guide my mouse around the perimeter if I want to click Back/Forward buttons for the tab on the right. And "effect" was correct btw. Cheers.

You can work around this by using the Alt + Left/Right Arrow keys shortcut.

**Comment by diego-velez-upr** [comment]
I'm having the same issue on Linux, it is always enabled for me no matter what.

**Comment by diego-velez** [comment]
I'm having the same issue on Linux, it is always enabled no matter what.

**Comment by rubaboo** [comment]
> You can work around this by using the Alt + Left/Right Arrow keys shortcut.

Well, I find myself using the back button's right-click menu rather often (to jump a few steps back), and unfortunately there is no keyboard shortcut for that. I created a feature request #1922.

**Comment by BrhmDev** [comment]
This is fixed in 1.0.1-a.8

**Comment by sjclayton** [comment]
@BrhmDev

Not fixed here.... now the setting does nothing either way, checked it does nothing and unchecked it does nothing.

Running1.0.1-a.8 on Linux

**Comment by BrhmDev** [comment]
> @BrhmDev
> 
> Not fixed here.... now the setting does nothing either way, checked it does nothing and unchecked it does nothing.
> 
> Running1.0.1-a.8 on Linux

You are right will be fixed with https://github.com/zen-browser/desktop/pull/2043, if you want to change it right now you can do so by changing the "zen.splitView.change-on-hover" pref in about:config
